conf/application.conf

```diff
@@ -80,11 +85,13 @@ offender.api.provider=${?OFFENDER_API_PROVIDER}
 prisoner.api.provider="legacy"
 prisoner.api.provider=${?PRISONER_API_PROVIDER}
 
-custody.api.token.cache.time.seconds=3540
-custody.api.auth.username=""
-custody.api.auth.username=${?CUSTODY_API_USERNAME}
-custody.api.auth.password=""
-custody.api.auth.password=${?CUSTODY_API_PASSWORD}
+hmpps.auth.token.cache.time.seconds=3540
+hmpps.auth.username=""
+hmpps.auth.username=${?CUSTODY_API_USERNAME}
```

Comment:
It would be nice to clean these environment variables to make them consistent, eg. should really be HMPSS_AUTH_CLIENT_ID etc